examples/Lorenz/Lorenz_example.jl

```diff
@@ -0,0 +1,398 @@
+# Import modules
+include("GModel.jl") # Contains Lorenz 96 source code
+
+include("/groups/esm/mhowland/Codes/CalibrateEmulateSample.jl/src/ParameterDistribution.jl")
```

Comment:
For testing, add `CalibrateEmulateSample` into the package manager by calling
`develop /groups/esm/mhowland/Codes/CalibrateEmulateSample.jl`
then you can use the commented statements below and remove these local includes

Reply:
Done, thanks!